File: problems/n-repeated_element_in_size_2n_array/solution.py
class Solution:
    def repeatedNTimes(self, nums: List[int]) -> int:
        lt = {}
        n=int(len(nums)/2);
        for i in nums:
            if i in lt:
                lt[i]+=1;
            else:
                lt[i] = 1;           
            if lt[i] == n:
                return i;
        return -1;

File: problems/n-repeated_element_in_size_2n_array/solution.cpp
class Solution {
public:
    int repeatedNTimes(vector<int>& nums) {
        unordered_map<int,int> map;
        int n=nums.size();
        n=n/2;
        for(auto &i:nums){
            map[i]++;
            if(map[i] >= n){
                return i;
            }
        }
        return -1;
    }
};